tests/src/io/test-save-geomodel.cpp

```diff
@@ -199,6 +199,7 @@ void io_geomodel(
 template < index_t DIMENSION >
 void process_extension( const std::string& extension )
 {
+#ifdef WIN32
```

Comment:
Why is there a difference on Windows?

Reply:
because it works under linux but not under window.... Cells are not printed following the same order

Reply:
Ok